morpheus/_lib/src/messages/meta.cpp

```diff
@@ -157,15 +158,35 @@ cudf::size_type MessageMetaInterfaceProxy::count(MessageMeta &self)
 
 pybind11::object MessageMetaInterfaceProxy::get_data_frame(MessageMeta &self)
 {
-    // // Get the column and convert to cudf
-    // auto py_table_struct = make_table_from_view_and_meta(self.m_pydf;.tbl->view(),
-    // self.m_pydf;.metadata); py::object py_table  =
-    // py::reinterpret_steal<py::object>((PyObject*)py_table_struct);
-
-    // // py_col.inc_ref();
+    // Release any GIL
+    pybind11::gil_scoped_release no_gil;
 
     // return py_table;
-    return self.get_py_table();
+    return self.get_info().copy_to_py_object();
```

Comment:
If we make this change, we should also make a copy in the Python impl of this class.

Otherwise we will get different behaviors in Python stages when C++ execution is enabled/disabled.